src/workflows/recipes/pages/RecipeDetailPage.js

```diff
@@ -19,7 +19,7 @@ import { getUrlParamAsInt } from 'console/state/router/selectors';
 @connect((state, props) => {
   const recipeId = getUrlParamAsInt(state, 'recipeId');
   const latestRevisionId = getLatestRevisionIdForRecipe(state, recipeId, '');
-  const revisionId = getUrlParamAsInt(state, 'revisionId', latestRevisionId);
+  const revisionId = getUrlParamAsInt(state, 'revisionId', latestRevisionId) || null;
```

Comment:
I have concerns about this line, since `getUrlParamAsInt` already takes a fallback value as its third item, and we are already providing that fallback in the form of `latestRevisionId`. Can you explain this a bit? Would it make sense to change the fallback value for `latestRevisionId` to `null` intead of `''`?

Reply:
Good catch! I think you're right. I'll fold it in tomorrow.